src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.commands;
+
+/**
+ * Sorts the address book alphabetically.
+ */
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = "Sorts the address book alphabetically.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book has been sorted!";
```

Comment:
Good, but if this constant is only going to be used internally, perhaps can use private accessor instead